RN_Components/src/components/play-pause/PlayPause.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+
+import { styles } from './playPause-styles';
+
+interface PlayPauseProps {
+  setPaused: React.Dispatch<React.SetStateAction<boolean>>;
+  paused: boolean;
+}
+
+const PlayPause = ({ setPaused, paused }) => {
+  const handlePlayPause = () => {
+    setPaused(!paused);
+  };
+
+  return (
+    <View>
+      <TouchableOpacity onPress={handlePlayPause}>
+        {paused ? (
```

Comment:
can use condition on name of icon only , we don't have to put it icon as everything else is same , just name is conditional